### PR TITLE
Delete "share screen", "start task comment on slack" on Issue and PR …

### LINKED ISSUE #66: Delete "share screen", "start task comment on slack" on Issue and PR template
# Tasks

Delete the below tasks from custom.md
- [x] Delete "Share your screen on Discord" 
- [x] Delete "Write which issue you are starting on Slack"

Delete  the below tasks from pull_request_template.md
- [x] Delete "Write which PR you are reviewing on Slack"

Estimated Time: XXmin

# Writing the Issue

- [x] Write each task for this issue using checkboxes
- [x] Write all verbal instructions as tasks
- [x] Add labels and assignees

# Before Working on the Issue

- [x] Assign yourself
- [x] Share your screen on Discord
- [x] Write which issue you are starting on Slack
- [x] Git: Fetch the latest main
- [x] Git: Create a branch with the name of the issue

# Pre-PR Checks

- [x] Insure that changes include only what is necessary for this PR
- [x] Implement necessary Unit tests
- [x] Implement necessary E2E tests
- [x] Perform functionality checks

### PR DESCRIPTION
**Pull request by REI-MORI**
…template #66

closes #66 

# For the Submitter

## Tasks

- [x] Make sure all checkboxes in the Issue content are checked
- [x] Make sure the PR title is in the format `<Issue Title> <Issue Number>`
- [x] Make sure "closes (issue number)" is written close to the top
- [x] Attach the enhancement label for update tasks
- [x] Make sure CI completes successfully
- [x] Check everything written in the PR
- [x] Fix the issues pointed out by AI
- [x] Add reviewers
- [x] On Slack, mention the PR reviewer, write your thoughts on the issue, and inform them that the PR has been created

# For the Reviewer

## Before Reviewing

- [x] Write which PR you are reviewing on Slack

## Checks

- [x] Each task for the related issue are written and in checkboxes
- [x] The branch name is close to the issue name
- [x] CI completes successfully
- [x] Changes include only what is necessary for this PR
- [x] Necessary unit tests are implemented
- [x] Necessary E2E tests are implemented
- [x] Perform functionality checks
- [x] If there are any problems, request revisions from the PR submitter and mention them on Slack

## After Reviewing

- [x] Leave an approval comment (e.g. LGTM)
- [x] On Slack, mention the PR submitter and a project admin, write which PR you have reviewed, and ask them to merge the PR
- [x] Write an approval comment with praise on Slack

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -18,8 +18,6 @@ closes #ISSUE_NUMBER
 
 ## Before Reviewing
```
Comment (Acha0203) [inline .github/pull_request_template.md:19]: [IMO]

見出しだけ残っていると変なので、この行も削除したほうが良いのではないでしょうか？

**Review by Acha0203 — COMMENTED** [review]
@REI-MORI 

1 件コメントしましたので、ご確認をお願いいたします。

File: .github/pull_request_template.md (hunk 1 above)
Comment (REI-MORI) [reply .github/pull_request_template.md]: @Acha0203 ご指摘ありがとうございます。修正いたしました。

**Review by Acha0203 — APPROVED** [review]
LGTM

**Comment by Acha0203** [comment]
@sinproject-iwasaki 

`npm install` したときに、critical な脆弱性が 1 件検出されましたので、ご報告いたします。

`npm audit fix` などで対応が必要な場合はご連絡ください。よろしくお願いいたします。

**Review by Acha0203 — APPROVED** [review]
LGTM

**Review by iamsin — APPROVED** [review]
LGTM

**Comment by iamsin** [comment]
@Acha0203 

別 Issue を作成して、npm audiit fix 対応を行ってください。
他に良い方法があればご提案くださいませ。

**Pull request merged**